### PR TITLE
Fix tox casing to be consistent with their docs, remove . from pytest

### PR DESCRIPTION
**Pull request by theacodes**
Ref: https://github.com/tox-dev/tox/issues/1088

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -39,9 +39,9 @@ For each session, Nox will automatically create `virtualenv`_ with the appropria
 
 To learn how to install and use Nox, see the :doc:`tutorial`. For documentation on configuring sessions, see :doc:`config`. For documentation on running ``nox``, see :doc:`usage`.
 
-.. _Tox: https://tox.readthedocs.org
+.. _tox: https://tox.readthedocs.org
```
Comment (dhermes) [inline docs/index.rst]: Sphinx / docutils / rST doesn't respect casing (not that it matters, and I also match casing)

Comment (theacodes) [reply docs/index.rst]: Interesting. :) Good to know, I'll do all of my links in TeEnAgEr CaSe from now on.

Comment (SethMichaelLarson) [reply docs/index.rst]: [mOcKiNg SpOnGeBoB cAsInG](https://i.kym-cdn.com/entries/icons/original/000/022/940/mockingspongebobbb.jpg)

**Comment by obestwalter** [comment]
Thanks @theacodes!

**Pull request merged**